packages/mermaid/src/docs/.vitepress/config.ts

```diff
@@ -4,7 +4,7 @@ import { defineConfig, MarkdownOptions } from 'vitepress';
 
 const allMarkdownTransformers: MarkdownOptions = {
   // the shiki theme to highlight code blocks
-  theme: 'github-dark',
+  theme: 'github-light',
   config: async (md) => {
```

Comment:
Changing this will break the site in dark mode. We should set the value based on the dark/light mode that the site is using. You can refer the vitepress documentation to figure out how this can be done.

Reply:
Thanks @sidharthv96 I just found the option and updated the PR :)